examples/svelte/basic/src/App.svelte

```diff
@@ -0,0 +1,143 @@
+<script lang="ts">
+  import './index.css'
+  import {
+    createTable,
+    getCoreRowModelSync,
+    createTableInstance,
+  } from '@tanstack/svelte-table'
+
+  type Person = {
+    firstName: string
+    lastName: string
+    age: number
+    visits: number
+    status: string
+    progress: number
+  }
+
+  const table = createTable().setRowType<Person>()
+
+  const defaultData: Person[] = [
+    {
+      firstName: 'tanner',
+      lastName: 'linsley',
+      age: 24,
+      visits: 100,
+      status: 'In Relationship',
+      progress: 50,
+    },
+    {
+      firstName: 'tandy',
+      lastName: 'miller',
+      age: 40,
+      visits: 40,
+      status: 'Single',
+      progress: 80,
+    },
+    {
+      firstName: 'joe',
+      lastName: 'dirte',
+      age: 45,
+      visits: 20,
+      status: 'Complicated',
+      progress: 10,
+    },
+  ]
+
+  const defaultColumns = [
+    table.createGroup({
+      header: 'Name',
+      footer: props => props.column.id,
+      columns: [
+        table.createDataColumn('firstName', {
+          cell: info => info.value,
+          footer: props => props.column.id,
+        }),
+        table.createDataColumn(row => row.lastName, {
+          id: 'lastName',
+          cell: info => info.value,
+          header: () => 'Last Name',
+          footer: props => props.column.id,
+        }),
+      ],
+    }),
+    table.createGroup({
+      header: 'Info',
+      footer: props => props.column.id,
+      columns: [
+        table.createDataColumn('age', {
+          header: () => 'Age',
+          footer: props => props.column.id,
+        }),
+        table.createGroup({
+          header: 'More Info',
+          columns: [
+            table.createDataColumn('visits', {
+              header: () => 'Visits',
+              footer: props => props.column.id,
+            }),
+            table.createDataColumn('status', {
+              header: 'Status',
+              footer: props => props.column.id,
+            }),
+            table.createDataColumn('progress', {
+              header: 'Profile Progress',
+              footer: props => props.column.id,
+            }),
+          ],
+        }),
+      ],
+    }),
+  ]
+
+  let data = defaultData
+  let columns = defaultColumns
+
+  const rerender = () => {
+    data = data
+  }
+
+  $: instance = createTableInstance(table, {
+    data,
+    columns,
+    getCoreRowModel: getCoreRowModelSync(),
+  })
```

Comment:
`instance` should be a normal variable and not a dependent one. Making it dependent would cause react-table to create a new instance each time any of the options like data and columns update which would cause it to lose its internal state. To make the options reactive you'll have to make them a svelte store